stubs/bleach/bleach/sanitizer.pyi

```diff
@@ -24,7 +24,7 @@ class NoCssSanitizerWarning(UserWarning): ...
 
 # A html5lib Filter class
 class _Filter(Protocol):
-    def __call__(self, *, source: BleachSanitizerFilter) -> BleachSanitizerFilter: ...
+    def __call__(self, *, source: BleachSanitizerFilter) -> Filter: ...
```

Comment:
Let's remove the obsolete comment and update the protocol name. Possibly something like `_CreateFilter`.

Reply:
Can we keep an alias to the old name so there's less disruption for people that are using the old name?

Reply:
If you are using the alias or know of people that are, that makes sense to me. (With a comment that this alias is obsolete) Otherwise I wouldn't bother, unless mypy primer complains about this.

We also need to add a `@type_check_only` decorator to the protocol.